From 111ed8a8fa0da001ef16c942d37edcef33f80805 Mon Sep 17 00:00:00 2001
From: Bryce Larson <blarson@saltstack.com>
Date: Wed, 23 Oct 2019 00:11:35 -0600
Subject: [PATCH 4/8] Add Feitian Technologies, Inc. ePass FIDO to devices

---
 u2flib_host/hid_transport.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/u2flib_host/hid_transport.py b/u2flib_host/hid_transport.py
index e16eb20..8b9a220 100644
--- a/u2flib_host/hid_transport.py
+++ b/u2flib_host/hid_transport.py
@@ -38,6 +38,7 @@ from u2flib_host.yubicommon.compat import byte2int, int2byte
 from u2flib_host import exc
 
 DEVICES = [
+    (0x096e, 0x0850),  # Feitian Technologies, Inc. ePass FIDO
     (0x1050, 0x0200),  # Gnubby
     (0x1050, 0x0113),  # YubiKey NEO U2F
     (0x1050, 0x0114),  # YubiKey NEO OTP+U2F
-- 
2.21.3

