From e92c3cbd840b651ed4b9c50a9797393f6d3b0c70 Mon Sep 17 00:00:00 2001
From: Saswat Padhi <padhi@cs.ucla.edu>
Date: Sun, 24 Nov 2019 03:33:51 -0800
Subject: [PATCH 6/8] Added OnlyKey device

---
 u2flib_host/hid_transport.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/u2flib_host/hid_transport.py b/u2flib_host/hid_transport.py
index eef954c..4014aac 100644
--- a/u2flib_host/hid_transport.py
+++ b/u2flib_host/hid_transport.py
@@ -55,6 +55,7 @@ DEVICES = [
     (0x096e, 0x085b),  # FS ePass FIDO
     (0x24dc, 0x0501),  # JaCarta U2F
     (0x1ea8, 0xf025),  # Thetis U2F
+    (0x1d50, 0x60fc),  # OnlyKey U2F
 ]
 HID_RPT_SIZE = 64
 
-- 
2.21.3

