From 11ddd13557f0fe8bfa09d5b54e2427204dd58b8e Mon Sep 17 00:00:00 2001
From: Dain Nilsson <dain@yubico.com>
Date: Thu, 5 Dec 2019 13:24:46 +0100
Subject: [PATCH 7/8] Add Trezor VID/PID (close #42).

---
 u2flib_host/hid_transport.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/u2flib_host/hid_transport.py b/u2flib_host/hid_transport.py
index 4014aac..f7f71ca 100644
--- a/u2flib_host/hid_transport.py
+++ b/u2flib_host/hid_transport.py
@@ -56,6 +56,7 @@ DEVICES = [
     (0x24dc, 0x0501),  # JaCarta U2F
     (0x1ea8, 0xf025),  # Thetis U2F
     (0x1d50, 0x60fc),  # OnlyKey U2F
+    (0x1209, 0x53c1),  # Trezor U2F/FIDO2
 ]
 HID_RPT_SIZE = 64
 
-- 
2.21.3

